api/controllers/auth.js

```diff
@@ -2,14 +2,16 @@
 // const models = require('../../database/models');
 const response = require('../helpers/response');
 const jwt = require('../helpers/jwt');
+const keys = require('../../config/secret');
 
 module.exports = {
   async socialAuthlogin(req, res, next) {
     const { user } = req;
 
     try {
       const token = await jwt.generateToken(user.dataValues);
-
+      console.log(req.headers.host);
+      res.redirect(`${keys.REDIRECT_URL}?token=${token}`);
```

Comment:
I'd suggest renaming `REDIRECT_URL` to a more explicit name, maybe "FRONTEND_BASE_URL"?.